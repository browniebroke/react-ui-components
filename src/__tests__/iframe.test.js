import React from 'react'
import renderer from 'react-test-renderer'
import { ThemeProvider } from 'styled-components'

import styledComponentsTheme from '../styled-components-theme'
import ResponsiveIframeWrapper from '../iframe'

describe('ResponsiveIframeWrapper component', () => {
  test('should render correctly', () => {
    let tree = renderer
      .create(
        <ThemeProvider theme={styledComponentsTheme}>
          <ResponsiveIframeWrapper yRatio={21} xRatio={9} />
        </ThemeProvider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
