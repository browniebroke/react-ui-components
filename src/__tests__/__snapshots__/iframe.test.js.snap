// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResponsiveIframeWrapper component should render correctly 1`] = `
<div
  className="sc-bdfBwQ kLFcVt"
/>
`;
